variables.tf

```
@@ -4,30 +4,67 @@
 # You must provide a value for each of these parameters.
 # ------------------------------------------------------------------------------
 
-variable "subnet_id" {
-  description = "The ID of the AWS subnet to deploy into (e.g. subnet-0123456789abcdef0)."
-  type        = string
-}
-
 # ------------------------------------------------------------------------------
 # OPTIONAL PARAMETERS
 #
 # These parameters have reasonable defaults.
 # ------------------------------------------------------------------------------
-variable "ami_owner_account_id" {
-  default     = "self"
-  description = "The ID of the AWS account that owns the Example AMI, or \"self\" if the AMI is owned by the same account as the provisioner."
+
+variable "aws_region" {
+  default     = "us-east-1"
+  description = "The AWS region to deploy into (e.g. us-east-1)."
   type        = string
 }
 
-variable "aws_availability_zone" {
-  default     = "a"
-  description = "The AWS availability zone to deploy into (e.g. a, b, c, etc.)."
+variable "expiration_days" {
+  default     = 45
+  description = "A strictly positive integer denoting the number of days after which an IAM user's access is considered inactive if unused."
+  type        = number
+
+  validation {
+    condition     = alltrue([floor(var.expiration_days) == var.expiration_days, var.expiration_days > 0])
+    error_message = "expiration_days must be a positive integer greater than zero."
+  }
+}
+
+variable "lambda_function_description" {
+  default     = "Lambda function to disable inactive IAM users."
+  description = "The description of the Lambda function."
```

Corresponding to a similar suggestion above:
```suggestion
  description = "The description of the Lambda function that will disable inactive IAM users."
```